src/components/ObservationsFlashList/ObsPressable.tsx

```diff
@@ -18,7 +18,6 @@ type Props = {
   onUploadButtonPress: ( ) => void,
   onItemPress: ( ) => void,
   gridItemStyle: Object,
-  isLargeFontScale: boolean,
```

Comment:
This is still used as a prop in `ObservationsFlashList` and should be removed there.